src/gui/transferlistmodel.cpp

```diff
@@ -491,6 +501,8 @@ QVariant TransferListModel::internalValue(const BitTorrent::Torrent *torrent, co
         return torrent->remainingSize();
     case TR_TIME_ELAPSED:
         return !alt ? torrent->activeTime() : torrent->finishedTime();
+    case TR_DOWNLOAD_PATH:
+        return torrent->downloadPath().toString();
```

Comment:
Since this value is internal:
```suggestion
        return torrent->downloadPath().data();
```

Reply:
What do you mean by "internal"? Both `Bittorrent::savePath` and `Bittorrrent::DownloadPath` return `Path` object. Why should savePath use `toString()` and downloadPath `data()`?

Reply:
>What do you mean by "internal"? 

Here it is passing a value to the internal storage of the transfer list model. And this internal storage is mainly used for sorting as you can see.

>Both Bittorrent::Torrent::savePath and Bittorrrent::Torrent::DownloadPath return Path object. Why should savePath use toString() and downloadPath data()?

I reckon both of them should use `data()`. IMO this was neglected when introducing the Path type. Anyway you can ignore `savePath` and I'll change it later.

Reply:
Ok, `data()` returns a pointer to the internal data (the chars) wereas `toString()` would return a new `QString` object.